media-proxy/src/libfabric_dev.c

```diff
@@ -102,17 +102,17 @@ int rdma_init(libfabric_ctx **ctx)
         return -ENOMEM;
     }
 
-    hints->fabric_attr->prov_name = strdup("verbs");
-
-    hints->caps = FI_MSG;
-    hints->domain_attr->resource_mgmt = FI_RM_ENABLED; /* TODO: check performance */
-    hints->mode = FI_CONTEXT;
-    hints->domain_attr->threading = FI_THREAD_SAFE; /* TODO: check performance */
-    hints->addr_format = FI_FORMAT_UNSPEC;
-    hints->ep_attr->type = FI_EP_RDM;
     hints->domain_attr->mr_mode =
         FI_MR_LOCAL | FI_MR_ENDPOINT | (FI_MR_ALLOCATED | FI_MR_PROV_KEY | FI_MR_VIRT_ADDR);
+    hints->ep_attr->type = FI_EP_RDM;
+    hints->caps = FI_MSG;
+    hints->addr_format = FI_SOCKADDR_IN;
+    hints->fabric_attr->prov_name = strdup("tcp");
```

Comment:
Consider using some predefined global const for that like below
```c++
const char * LIB_FABRIC_ATTR_PROV_NAME_TCP = "tcp";
```

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/293/commits/907a4e3917a16fa35b4ebde105a15f179531b040